ptbcontrib/chat_to_link/__init__.py

```diff
@@ -0,0 +1,21 @@
+# A library containing community-based extension for the python-telegram-bot library
+# Copyright (C) 2020-2021
+# The ptbcontrib developers
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser Public License for more details.
+#
+# You should have received a copy of the GNU Lesser Public License
+# along with this program.  If not, see [http://www.gnu.org/licenses/].
+"""This module contains a helper function to get joinable links from chats."""
```

Comment:
no need of this here

Reply:
I added it because the other features (e.g. [`extract_urls.__init__`](https://github.com/python-telegram-bot/ptbcontrib/blob/main/ptbcontrib/extract_urls/__init__.py#L17)) had them. Should I remove it?

Reply:
Hm, then I guess it'd be ok for this to be there.